opendevin/storage/s3.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from minio import Minio
+
+from .files import FileStore
+
+AWS_S3_ENDPOINT = 's3.amazonaws.com'
+
+
+class S3FileStore(FileStore):
```

Comment:
I think maybe put s3.py into a directory of cloud provider will be better, so we can add some other cloud file storage like azure bolb, Google Cloud Storage.

Reply:
agree we'll probably add these in the future! Let's create a subdirectory then if we decide we need it